src/app/auth/register/register.component.ts

```diff
@@ -59,29 +56,13 @@ export class RegisterComponent implements OnInit, DoCheck {
     return email === confirmEmail ? null : { EmailMismatch: true }
   }
 
-  public ngOnInit(): void {}
-
-  ngDoCheck(): void {
-
-    //On email change
-    if (this.registerForm.get('email')?.value.toLowerCase() != this.oldEmail){
-
-      this.oldEmail = this.registerForm.get('email')?.value.toLowerCase();
-      this.registerForm.controls['displayName'].setValidators(
-        Validators.compose([Validators.required, notEmailValidator(this.registerForm.get('email')?.value.split("@", 2)[0].toLowerCase())])
-      )
-      if (this.registerForm.get('email')?.value.split("@", 2)[0].toLowerCase() === this.registerForm.get('displayName')?.value.toLowerCase()){
-        this.registerForm.controls['displayName'].reset();
-        this.registerForm.controls['displayName'].setValue(this.registerForm.get('email')?.value.split("@", 2)[0]);
-      }
-      this.uniqueEmailFlag = true;
-    }
-
-    //On Display Name change
-    if (this.registerForm.get('displayName')?.value != this.oldDisplayName){
-      this.oldDisplayName = this.registerForm.get('displayName')?.value;
-      this.uniqueDisplayNameFlag = true;
-    }
+  /**
+   * Validates that display name is not email handle, case insensitive
+   */
+   notEmailHandleValidator: ValidatorFn = (group: AbstractControl):  ValidationErrors | null => {
```

Comment:
Display name can be the user's email.

Reply:
I had understood that it did need to be different
![registerMemorandum](https://user-images.githubusercontent.com/23206954/172979209-c7b312e4-e0a6-4537-ab77-8dd056eb050c.PNG)

Reply:
Whoops. You're right.